contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -43,6 +43,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   mapping(uint256 => Upkeep) internal s_upkeep;
   mapping(address => KeeperInfo) internal s_keeperInfo;
   mapping(address => address) internal s_proposedPayee;
+  mapping(uint256 => address) internal s_proposedAdmin;
```

Comment:
We'll also need to handle this in migration. (Can be a separate PR)

Reply:
i think this can be done by `delete s_proposedAdmin[id];` in `migrateUpkeep` ?

Reply:
yes i think that will work, much simpler